client/src/components/manageProjects/editProject.js

```diff
@@ -24,6 +24,12 @@ const EditProject = ({
   const [selectedEvent, setSelectedEvent] = useState();
   const [isCreateNew, setIsCreateNew] = useState();
 
+  // States for alerts
+  const [showNewEventCreatedAlert, setShowNewEventCreatedAlert] = useState(
```

Comment:
What if we just have a single alert state? `const [eventAlert, setEventAlert] = useState(null);` You can then just call setEventAlert('Event Updated!') from the child components with the alert text.